src/styles/_mixin.scss

```diff
@@ -0,0 +1,6 @@
+@mixin box-shadow($x-axis: 0, $y-axis: 1px, $blur: 4px, $color: $default) {
```

Comment:
I guess longer term Tim doesn't want CSS in the data browser, but if we do, we should probably use something like [autoprefixer](https://www.npmjs.com/package/autoprefixer) instead.